phpstan-baseline.neon

```diff
@@ -0,0 +1,31 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Variable \\$value on left side of \\?\\? always exists and is always null\\.$#"
+			count: 1
+			path: src/Eloquent/Model.php
+
+		-
+			message: "#^Variable \\$value on left side of \\?\\? always exists and is not nullable\\.$#"
+			count: 1
+			path: src/Eloquent/Model.php
+
+		-
+			message: "#^Cannot call method modelKeys\\(\\) on array\\|Illuminate\\\\Support\\\\Collection\\.$#"
+			count: 1
+			path: src/Relations/BelongsToMany.php
+
+		-
+			message: "#^Method Illuminate\\\\Database\\\\Eloquent\\\\Model\\:\\:push\\(\\) invoked with 3 parameters, 0 required\\.$#"
```

Comment:
The `push()` method uses `func_get_args()` to get its arguments, but we're passing it to the method as arguments. Not sure how to fix or write differently.

Reply:
My idea was to add phpdoc for this arguments, but PHPStorm and phpcs both complain that "PHPDoc tag @param references unknown parameter". 

```
    /*
     * If no arguments are provided, save the model and all of its relationships.
     * If a $column is provided, append the $values to the $column array attribute.
     *
     * @param string $column
     * @param mixed  $values
     * @param bool   $unique
     *
     * @return bool
     */
    public function push(/* string $column = null, $values = null, bool $unique = false */)
```

I'm delaying it for a later PR. No doubt the design of this method is incorrect and needs to be changed.